_includes/events-posts.html

```diff
@@ -0,0 +1,22 @@
+
+{% macro render(posts) %}
+
+{% import "event-post-macros.html" as event_post_macros with context %}
+
+{% for post in posts%}
+
+    <article class="post-preview event-post-preview">
```

Comment:
`event-post-prevew` should be `post-preview__event` to follow the [CFPB flavor of BEM](http://cfpb.github.io/capital-framework/contributing/#follow-our-css-naming-conventions).